manager/src/main/java/com/redhat/service/smartevents/manager/ProcessorService.java

```diff
@@ -25,6 +27,8 @@ public interface ProcessorService {
 
     ListResult<Processor> getProcessors(String bridgeId, String customerId, QueryProcessorResourceInfo queryInfo);
 
+    ListResult<Processor> getAllProcessors(String bridgeId, String customerId);
```

Comment:
The changes to `BridgeWorker` (where the Error Handling `Processor` is created) needs to get all `Processor`'s whether in `READY` state or not. `BridgeWorker` checks if an Error Handling `Processor` has already been requested to ensure the creation of one is idempotent since `BridgeWorker` is invoked successive times when provisioning a `Bridge`.